### PR TITLE
Add codemod and tests for the Avatar component rename

### PR DESCRIPTION
**Pull request by lavelle**

### CODE REVIEW

File: packages/thumbprint-codemods/src/avatar-import-name/test.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            transform(`
+import React from 'react';
+import { Avatar, Button, Input } from '@thumbtack/thumbprint-react';
+const App = (props) => (
+    <div>
+        <Avatar />
+        <Button />
+        <Input />
+    </div>
+ )`),
```
Comment (danoc) [inline packages/thumbprint-codemods/src/avatar-import-name/test.js]:
```suggestion
)`),
```

Comment (lavelle) [reply packages/thumbprint-codemods/src/avatar-import-name/test.js]: done

**Comment by lavelle** [comment]
> This is great. Their utility function makes it so easy.
> 
> Does this work well if:
> 
> * `@thumbtack/thumbprint-react` is not one of the imports?
> * `@thumbtack/thumbprint-react` is an import but `Avatar` is not imported?

Could you give an example of these tests? Surely if we're not importing `Avatar` then the code should be a no-op, right?

**Comment by danoc** [comment]
@lavelle — Just pushed a failing test case.

This code:

```jsx
import React from 'react';
import Avatar from '../my-avatar.jsx';
const App = (props) => (
    <Avatar />
)`;
```

...is transformed to:
import React from 'react';
import UserAvatar from '../my-avatar.jsx';
const App = (props) => (
    <UserAvatar />
)`;
```jsx

```

**Pull request merged**